### ISSUE TITLE
when will the new version 3.1.1 be released?

### ISSUE THREAD

**Issue by AndreaGProg** [froala/wysiwyg-editor#3865]
Hello,
Update expected soon?

3.1.0
https://froala.com/wysiwyg-editor/changelog/

3.1.1
https://libraries.io/npm/froala-editor/3.1.1

**Comment by chrisdeeming** [comment]
Have they even been working on it?

The level of output and support has been declining for months and it is really alarming. I'm still waiting for 2.9.7 which has been promised since September last year.

What are we paying for, exactly?

**Comment by AndreaGProg** [comment]
Version 3 has many bugs.
Even though I paid version 3, I still use version 2.9.6.

**Comment by chrisdeeming** [comment]
2.9.6 has many bugs too which have been fixed already in 2.9.7 but Froala seem unwilling or unable to release it. Shambolic.

**Comment by gabaum10** [comment]
I agree.  This is unacceptable.  We want to use version 3.0 because it has some nice features we built around (dynamic toolbar options).  However, it's so incredibly buggy, and there hasn't been any activity on this in months.

**Comment by chrisdeeming** [comment]
@froala-bot 

It has been nearly 1 year since 2.9.6 was released and we're still waiting for 2.9.7.

Why are you working on 3.2 (which is presumably a feature release) when it is clear that many need you to be focusing on quality and stability for the existing 3.1 release?

When will 3.1.1 be released? Why does there appear to be no activity on this repository in terms of commits and issues?

**Comment by JaxonWright** [comment]
It definitely seems like each build is way too big, causing releases to be infrequent even when critical bugs have been fixed in the new build for months. The newest version of v3 (3.1) is virtually unusable on IE11, which is critical.

**Comment by JaxonWright** [comment]
Apparently this just released yesterday. Their changelog incorrectly replaces 3.1.0's listing, with a release of Jan 13, 2020. It lists 2 changes with a 3rd "performance improvements" item.

**Comment by AndreaGProg** [comment]
2020-04-20: Version 3.1.1 is here.
But version 3.1.0 has disappeared.
https://froala.com/wysiwyg-editor/changelog/
I don't know if trying this version ... will it be stable?

**Comment by pleunv** [comment]
2.9.7 was recently also mistakenly released at "latest", be careful.

**Comment by ilyaskarim** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
